cps/pom.xml

```diff
@@ -77,9 +77,27 @@
             <artifactId>groovy-cps</artifactId>
             <version>1.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.ui</groupId>
+            <artifactId>jquery-detached</artifactId>
+            <version>1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.ui</groupId>
+            <artifactId>ace-editor</artifactId>
+            <version>0.9</version>
+        </dependency>
     </dependencies>
     <!-- TODO currently job depends on tests from here; this dependency should probably be broken -->
```

Comment:
:ant: This comment should be moved next to the `maven-jar-plugin` execution it applies to.